code/game/machinery/computer/communications.dm

```diff
@@ -455,6 +455,8 @@
 			dat +=  dat2
 			popup.set_content(dat)
 			popup.open()
+			popup.set_content(dat)

```

Comment:
add a comment explaining why this is here so that someone doesn't come along later and remove it thinking it's an accidental duplication